src/platform/nrfconnect/ExternalFlashManager.h

```diff
@@ -31,9 +31,19 @@ class ExternalFlashManager
         SLEEP
     };
 
-    virtual ~ExternalFlashManager() {}
+    // Not copyable or movable
+    ExternalFlashManager(ExternalFlashManager &&)                  = delete;
+    ExternalFlashManager & operator=(const ExternalFlashManager &) = delete;
+    ExternalFlashManager(const ExternalFlashManager &)             = delete;
+    ExternalFlashManager & operator=(ExternalFlashManager &&)      = delete;
 
-    virtual void DoAction(Action aAction)
```

Comment:
Since we remove the possibility of overriding this functionality, maybe it's better to move ExternalFlashManager to the sdk-nrf scope as it assumes the external flash is using QSPI bus which is specific to our DKs. That might not always be the case in end products, so we could at least make it more transparent and application- scoped.

Reply:
Discussed offline- we've decided to keep the implementation as is for now as ExternalFlashManager is used in core as well. Created a follow-up task to investigate how we can make this modular and allow other implementations (like SPI): KRKNWK-19452